packages/vuetify/src/components/VDataTable/VSimpleTable.sass

```diff
@@ -106,6 +106,32 @@
           > th
             height: $data-table-dense-header-height
 
+.v-data-table--has-top
+  > .v-data-table__wrapper
+    tbody
```

Comment:
should use `>` all the way down so that it does not propagate on inner tables

Reply:
sorry, i fixed it by https://github.com/vuetifyjs/vuetify/pull/12553/commits/fb691e53602e591cb4113f015eecbedfb3ce4a2c